Assets/General/Scripts/Console/FreeLookCam/FreeLookCamController.cs

```diff
@@ -0,0 +1,74 @@
+/*****************************************************************************
+// Name: FreeLookCamController.CS
+// Author: Nabil Tagba
+// Overview: Handles movement for the free look cam
+*****************************************************************************/
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+/// <summary>
+/// Free look cam movement
+/// </summary>
+public class FreeLookCamController : MonoBehaviour
+{
+    private PlayerInputMap _playerInput;
+
+    [Header("Settings")]
+    [SerializeField] private float _speed;
+    [SerializeField] private float _lookSpeed;
+
+    [Header("References")]
+    [SerializeField] private Rigidbody _body;
+    [HideInInspector]
+    public Transform _cam;
+
+    /// <summary>
+    /// happens on awake
+    /// </summary>
+    private void Awake()
+    {
+        _playerInput = new PlayerInputMap();
+        _playerInput.Enable();
+        
+    }
+    /// <summary>
+    /// happens on start
+    /// </summary>
+    private void Start()
+    {
+        //assign cam
+        _cam = Camera.main.gameObject.transform;
+    }
+    /// <summary>
+    /// happens every frame
+    /// </summary>
+    private void Update()
```

Comment:
Personally would prefer this as a coroutine, but honestly the optimization of debug tools isn't too important as they are for testing.